ansible_base/authentication/authenticator_plugins/oidc.py

```diff
@@ -215,3 +215,8 @@ def extra_data(self, user, backend, response, *args, **kwargs):
 
     def get_user_groups(self, extra_groups=[]):
         return extra_groups
+
+    def oidc_config(self):
+        # This is a copy of super without caching, which avoids data
+        # from one OIDC based authenticator showing up in another
+        return self.get_json(self.oidc_endpoint() + "/.well-known/openid-configuration")
```

Comment:
If you call the oidc_config from super would it still be cached? If not it would be nice to just `return super().oidc_config` or whatever.

Reply:
Yeah that doesn't work to fix the issue, guess the caching works in that case.